src/screens/ShipItemDetails/index.tsx

```diff
@@ -145,53 +153,38 @@ const ShipItemDetails = () => {
           </View>
         </View>
       </View>
-      <Text>{'Container'}</Text>
-      <AutoInputInternalLocation
-        label="AutoInputInternalContainer"
-        data={state.containerList ? state.containerList : []}
-        selectedContainerItem={selectedContainerItem}
-        selectedData={(selectedItem: any, index: number) => {
-          setSelectedContainerItem(index);
-          state.containerId = selectedItem;
-          const container = state.shipmentDetails.availableContainers[index];
-          setState({
-            ...state,
-            containerId: selectedItem,
-            container: container,
-            selectedContainerItem: index,
-          });
-        }}
-      />
+      <View style={styles.containerField}>
+        <Text>{'Container'}</Text>
+        <AutoInputInternalLocation
+          label="AutoInputInternalContainer"
+          data={state.containerList ? state.containerList : []}
+          selectedContainerItem={selectedContainerItem}
+          selectedData={(selectedItem: any, index: number) => {
+            setSelectedContainerItem(index);
+            state.containerId = selectedItem;
+            const container = state.shipmentDetails.availableContainers[index];
+            setState({
+              ...state,
+              containerId: selectedItem,
+              container: container,
+              selectedContainerItem: index,
+            });
+          }}
+        />
+      </View>
       <InputSpinner
         title={'Quantity to Pick'}
         value={state.quantityPicked}
         setValue={quantityPickedChange}
       />
       <View style={styles.bottom}>
         <Button
+          disabled={state?.container?.id === null || Number(state.quantityPicked) === 0}
```

Comment:
Use `!state?.container?.id` instead of `state?.container?.id === null`

Reply:
Use `Number(state.quantityPicked) <= 0` instead of `Number(state.quantityPicked) === 0`